alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -0,0 +1,83 @@
+# PrometheusDuplicateTimestamps
+
+## Meaning
+
+The `PrometheusDuplicateTimestamps` alert is triggered when there is a constant
+increase in dropped samples due to them having identical timestamps.
+
+## Impact
+
+Unwanted samples might be scraped, and desired samples might be dropped.
+Consequently, queries may yield incorrect and unexpected results.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+   for default cluster monitoring or for the instance that monitors user-defined
+   projects by viewing the alert message's `namespace` label: the namespace for
```

Comment:
Not a blocker since the wording comes from the `PrometheusRemoteStorageFailures` runbook IIUC ;) but I think that we should use consistent wording across all runbooks to name the "platform monitoring stack" and "user-defined monitoring stack".

Reply:
yep, let's create an issue for this, I'll be happy to take care of it.